packages/core/test/emulate/mockFetch.ts

```diff
@@ -14,7 +14,7 @@ export function mockFetch() {
     }
   }
 
-  window.fetch = (() => {
+  window.fetch = ((...[, init]) => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Just out of curiosity, do you know how this is transpiled in ES5? :)

Reply:
https://www.typescriptlang.org/play/?target=0#code/O4SwdgJg9sB0BmBTALgYwBYAIC8mAUesRA2gDSbgjIC6AlDgHyYDeAvrUA

Though, this syntax is a bit surprising :D 

Suggestion:

```ts
window.fetch = (_input, init) => {
  ..
}
```

Reply:
It makes sense when you think about it, but I prefer benoit's suggestion (so I don't have to think about it 😄)

Or event `(input, init) => {` with [args-after-used](https://eslint.org/docs/latest/rules/no-unused-vars#args-after-used)